mobile/src/main/java/ch/epfl/sdp/mobile/ui/play/PlayScreen.kt

```diff
@@ -24,8 +31,30 @@ import ch.epfl.sdp.mobile.ui.PawniesIcons
 fun PlayScreen(
     state: PlayScreenState,
     modifier: Modifier = Modifier,
-    contentPadding: PaddingValues = PaddingValues(),
+    contentPadding: PaddingValues = PaddingValues()
 ) {
+  val lazyColumnState = rememberLazyListState()
```

Comment:
You don't seem to be using this hoisted state except for passing it to the `LazyColumn`, so I suggest you don't explicitly manage it.